theme/static/js/hs-file-icons.js

```diff
@@ -29,9 +29,10 @@ function getFolderIcons() {
         "alt='Geographic Feature Aggregation Icon'/></span>";
 
     folderIcons.DEFAULT =
-        "<span class='fb-aggregation-icon'>" +
-        "<img src='/static/img/resource-icons/composite48x48.png' " +
-        "alt='Default Aggregation Icon'/></span>";
+        "<span class='fa-stack fb-stack'>" +
+        "<i class='fb-file-icon fa fa-folder fa-stack-2x fb-folder-aggregation' aria-hidden='true'></i>" +
```

Comment:
recommend use <em> instead of <i>
In HTML 4.01, the <i> tag was used to render text in italics. However, this is not necessarily the case with HTML5. Style sheets can be used to format the text inside the <i> element.

Reply:
@Maurier Do you know what I need to change here? This is your code for the file set icon.